src/ex1/index.html

```diff
@@ -1,12 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>Exercise 1</title>
+    <title>Todo App</title>
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-
-<h1>Todo list</h1>
+<div class="todoAppContainer">
+    <h1 class="titleH1">Todo list </h1>
```

Comment:
what if you'll change it later to `<h2>` or even to a div. would you change the classname as well?